### PR TITLE
project-labyrinth Jacob & Jennifer

### PR DESCRIPTION
**Pull request by JaEngd**

### CODE REVIEW

File: code/src/components/GameScreen.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+`
+const EndText = styled.p`
+  color: white;
+`
+
+const GameScreen = () => {
+    const dispatch = useDispatch();
+    let [direction, setDirection] = useState('');
+
+    const directionArray = [
```
Comment (rawisou) [inline code/src/components/GameScreen.js:45]: Interesting with this local stage! I notice that you also have this directionArray in reducers line 35 - 40. Wondering if that could just have been exported instead.

File: code/src/components/StartScreen.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    const onUserSet = () => {
+        dispatch(games.actions.setUsername(inputValue));
+        dispatch(generateGame());
+    }
+
+  return (
+      <div>
+          <h1>The Labyrith game</h1>
+          <div style={{marginBottom: '10px'}}>
+          <InputField
```
Comment (rawisou) [inline code/src/components/StartScreen.js:38]: For accessibility reason, I'd add a <label> tag to the form input or title="" in the InputField

File: code/src/reducers/games.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            headers: {'Content-Type': 'application/json'}, 
+            body: JSON.stringify({username: getState().game.username})
+        })
+            .then(res => res.json())
+            .then(game => dispatch(games.actions.setGame(game)))
+            .finally(() => dispatch(loading.actions.setLoading(false)));
+    }
+}
+
+const directionArray = [
```
Comment (rawisou) [inline code/src/reducers/games.js:35]: Looks like you have this in the GameScreen component too. Maybe export this direction array and import it in the GameScreen component could work too.

**Review by rawisou — COMMENTED** [review]
Hi Jacob and Jennifer,

Your site works as it should--all the functionalities work.
I really like how your game looks and like that you used local state for direction. It gives possibility to do more things with them in the frontend for sure.

I've left some comments about potentially duplicate code and some accessibility point of view :) 

Great job this week and I hope you had fun building it! 

//Rawi

**Review by Nosslexa — COMMENTED** [review]
Great job with this weeks project! Your site works as it should and i like your retro styling with styled components. As Rawi already noticed you have the directionArray in two places maybe this could be avoided. You fulfill all the requirements for this week, well done.

**Pull request closed without merging**